repos/system_upgrade/el7toel8/actors/checkbrltty/tests/component_test.py

```diff
@@ -0,0 +1,64 @@
+from leapp.snactor.fixture import current_actor_context
+from leapp.models import InstalledRedHatSignedRPM, RPM, BrlttyMigrationDecision
+from leapp.reporting import Report
+from leapp.libraries.actor.library import BrlttyConf
+import pytest
```

Comment:
Please mind import order
```
import os 

import pytest

import leapp.libraries
import leapp.models
...
```
https://github.com/oamg/leapp-guidelines/blob/master/python-coding-guidelines.md#5-avoid--imports-and-sort-imports-alphabetically

Reply:
Fixed.